packages/integration_test/README.md

```diff
@@ -1,16 +1,20 @@
-# integration_test (deprecated)
+# integration_test (moved)
 
-## DEPRECATED
+## MOVED
 
-This package has been moved to the Flutter SDK. Starting with Flutter 2.0,
-it should be included as:
+This package has been [moved to the Flutter
```

Comment:
I'm not convinced this is clearer; "moved" isn't generally a meaningful term for clients of a package (since where the source lives is generally irrelevant). The version on pub, where users are likely to be reading this, is deprecated.

Reply:
The version on Pub is deprecated, but the package isn't. We don't want to give the impression that integration testing should be done some other way. @sfshaza2 do you have any input here?

Reply:
I made some suggested changes to the text.